packages/hubble-idl/src/borrowing.json

```diff
@@ -3720,6 +4195,48 @@
         ]
       }
     },
+    {
+      "name": "WithdrawalCaps",
+      "type": {
+        "kind": "struct",
+        "fields": [
+          {
+            "name": "configCapacity",
```

Comment:
Are these `i64` values being mapped to `BNs` and then to `Decimals` correctly?

Reply:
Yup, they are!